### ISSUE TITLE
imeta add fails if key already exists

### ISSUE THREAD

**Issue by mrmh2** [jic-dtool/dtool-irods#1]
When adding files to a protodataset, setting the handle using imeta add fails if the handle already exists (so it isn't possible to overwrite):

```
WARNING:dtool_irods:Command failed: ['imeta', 'add', '-d', '/jic_archive/2707da2b-b0ca-4f48-b822-3f0d1fe70046/data/f10e10505676569214979e69714e364ee4cdca77', 'handle', u'Gw2_P_Rep1_T2/191_1.3.tif']
WARNING:dtool_irods:remote addresses: 149.155.213.138 ERROR: rcModAVUMetadata failed with error -809000 CATALOG_ALREADY_HAS_ITEM_BY_THAT_NAME

remote addresses: 149.155.213.138 ERROR: rcModAVUMetadata failed with error -809000 CATALOG_ALREADY_HAS_ITEM_BY_THAT_NAME
```

**Comment by mrmh2** [comment]
imeta set seems like it should do what we want instead of imeta add

**Comment by tjelvar-olsson** [comment]
Fixed in 46febf5
